src/pepr/operator/enque.ts

```diff
@@ -0,0 +1,68 @@
+import { Log } from "pepr";
```

Comment:
nit: shouldnt this file be enqueue.ts? enque is ink https://en.wiktionary.org/wiki/enque